cli/diff.rs

```diff
@@ -27,146 +165,60 @@ fn fmt_rem_text_highlight(x: &str) -> String {
   colors::white_on_red(x).to_string()
 }
 
-fn write_line_diff(
-  diff: &mut String,
-  orig_line: &mut usize,
-  edit_line: &mut usize,
-  line_number_width: usize,
-  orig: &mut String,
-  edit: &mut String,
-) {
-  let split = orig.split('\n').enumerate();
-  for (i, s) in split {
-    diff.push_str(&format!(
-      "{:width$}{} ",
-      *orig_line + i,
-      colors::gray(" |"),
-      width = line_number_width
-    ));
-    diff.push_str(&fmt_rem());
-    diff.push_str(s);
-    diff.push('\n');
-  }
-
-  let split = edit.split('\n').enumerate();
-  for (i, s) in split {
-    diff.push_str(&format!(
-      "{:width$}{} ",
-      *edit_line + i,
-      colors::gray(" |"),
-      width = line_number_width
-    ));
-    diff.push_str(&fmt_add());
-    diff.push_str(s);
-    diff.push('\n');
-  }
-
-  *orig_line += orig.split('\n').count();
-  *edit_line += edit.split('\n').count();
-
-  orig.clear();
-  edit.clear();
-}
-
-/// Print diff of the same file_path, before and after formatting.
-///
-/// Diff format is loosely based on Github diff formatting.
-pub fn diff(orig_text: &str, edit_text: &str) -> String {
-  let lines = edit_text.split('\n').count();
-  let line_number_width = lines.to_string().chars().count();
-
-  let mut diff = String::new();
-
-  let mut text1 = orig_text.to_string();
-  let mut text2 = edit_text.to_string();
-
-  if !text1.ends_with('\n') {
-    text1.push('\n');
-  }
-  if !text2.ends_with('\n') {
-    text2.push('\n');
-  }
```

Comment:
Bug here. Should not do this because it should display a diff for when one has a trailing newline and the other doesn't.